libopae/plugins/xfpga/error.c

```diff
@@ -98,10 +98,15 @@ fpga_result __FPGA_API__ xfpga_fpgaClearError(fpga_token token, uint32_t error_n
 				FPGA_MSG("can't clear error '%s'", p->info.name);
 				return FPGA_NOT_SUPPORTED;
 			}
-			// read current error value
-			res = xfpga_fpgaReadError(token, error_num, &value);
-			if (res != FPGA_OK)
-				return res;
+
+			if (strcmp(p->info.name, "inject_error") == 0) {
```

Comment:
#define inject_err string

Reply:
use strcmp_s

Reply:
strcmp is not banned:
https://github.com/intel/safestringlib/wiki/SDL-List-of-Banned-Functions